conf/broker.conf

```diff
@@ -148,6 +148,15 @@ maxUnackedMessagesPerBroker=0
 # limit/2 messages
 maxUnackedMessagesPerSubscriptionOnBrokerBlocked=0.16
 
+# Subscribe too much for a consumer and load data from bookie to cause the high network bandwidth usage.
+# https://github.com/apache/pulsar/issues/2876
+# Default broker do not limit subscribe times for a consumer. Using a value of 0 is disabling subscribe
```

Comment:
When the positive value is set, broker will throttle the subscribe requests for one consumer. Otherwise, the throttling will be disabled. The default value of this setting is 0 - throttling is disabled.